corehq/apps/consumer_user/tests/test_views.py

```diff
@@ -0,0 +1,333 @@
+from django.test import TestCase
+from django.test.client import Client
+from corehq.apps.consumer_user.models import ConsumerUserInvitation
+from django.urls import reverse
+from django.contrib.auth.models import User
+from corehq.apps.consumer_user.models import ConsumerUser
+from corehq.apps.consumer_user.models import ConsumerUserCaseRelationship
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from mock import patch
+from corehq.apps.hqcase.utils import update_case
+from django.core.signing import TimestampSigner
+from corehq.apps.data_interfaces.tests.util import create_case
+from django.test.utils import override_settings
+
+
+def register_url(invitation):
+    return reverse('consumer_user:patient_register',
+                   kwargs={'invitation': TimestampSigner().sign(
+                       urlsafe_base64_encode(force_bytes(invitation))
+                   )})
+
+
+def login_accept_url(invitation):
+    return reverse('consumer_user:patient_login_with_invitation',
+                   kwargs={'invitation': TimestampSigner().sign(
+                       urlsafe_base64_encode(force_bytes(invitation))
+                   )})
+
+
+class RegisterTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.login_url = reverse('consumer_user:patient_login')
+        self.client = Client()
+
+    def test_register_get(self):
+        invitation = ConsumerUserInvitation.objects.create(case_id='1', domain='1', invited_by='I',
+                                                           invited_on='2021-02-09 17:17:32.229524+00',
```

Comment:
I don't think you need `invited_on` since it uses `auto_now`

Reply:
Fixed it